pkg/util/etcdutil/start_etcd.go

```diff
@@ -46,7 +46,7 @@ spec:
   containers:
   - command:
     - /bin/sh
-    - -c
+    - -ec
```

Comment:
Just my pick. I usually wrap everything in `sh -ec`. Then the whole command could be run in a single string.

I can change if you want.